.github/workflows/main.yml

```diff
@@ -118,6 +120,37 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install tox
-    - name: Test
-      run: tox -e ${{ matrix.tox_env }}
+        pip install tox coverage
+
+    - name: Test without coverage
+      if: "matrix.skip_coverage"
+      run: "tox -e ${{ matrix.tox_env }}"
+
+    - name: Test with coverage
+      if: "! matrix.skip_coverage"
+      env:
+        _PYTEST_TOX_COVERAGE_RUN: "coverage run -m"
+        COVERAGE_PROCESS_START: ".coveragerc"
+        _PYTEST_TOX_EXTRA_DEP: "coverage-enable-subprocess"
+      run: "tox -e ${{ matrix.tox_env }}"
+
+    - name: Prepare coverage token
+      if: success() && !matrix.skip_coverage && ( github.repository == 'pytest-dev/pytest' || github.event_name == 'pull_request' )
```

Comment:
@hugovk small difference here: we still upload coverage on PRs. Of course this may happen on PRs on fork -> fork, but this should be rare and not a problem.

Reply:
OK, should be fine.

So, when `github.event_name == 'pull_request'`, is it something like `github.repository == 'nicoddemus/pytest'`?

Reply:
Didn't check, but I think so